packages/@o3r/components/builders/component-extractor/index.ts

```diff
@@ -205,7 +205,8 @@ export default createBuilder(createBuilderWithMetricsIfInstalled<ComponentExtrac
     // Exit on watcher failure
     return new Promise<BuilderOutput>((_resolve, reject) =>
       watcher
-        .on('error', (err) => reject(err))
+        // remove cast after https://github.com/paulmillr/chokidar/issues/1392
```

Comment:
```suggestion
        // TODO remove cast after https://github.com/paulmillr/chokidar/issues/1392
```
or
```suggestion
        // FIXME remove cast after https://github.com/paulmillr/chokidar/issues/1392
```
or you can create a local issue to track this